mitodl/ocw-hugo-themes
# Update to Node v20.8.1

# What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/1263.

# Description (What does it do?)
This PR updates the version of Node to v20.8.1 to match the course publisher. It also updates the GitHub action workflows to use the same version.

# How can this be tested?
Assuming that [nvm](https://github.com/nvm-sh/nvm) is installed on your system, run the following:
1. `nvm install v20.8.1` to install the updated version.
2. `nvm use` to use it in the project.
3. `npm install -g yarn` to install `yarn` for this Node version.
4. `yarn start course <course-id>` for any course of your choosing, and verify that everything looks correct.
5. `yarn start www` and verify that everything looks correct.
